### ISSUE TITLE
Feature Request: Change the throughput test to use NDT

### ISSUE THREAD

**Issue by aww-aww** [Mobiperf/MobiPerf#136]
http://code.google.com/p/ndt/

**Comment by ghost** [comment]
I don't agree. We have NDT for android and having an alternate methodology that can be compared to the results from that mobile app is useful. Having two apps that measure the same thing is less useful from a research point of view.

**Comment by aww-aww** [comment]
How robust is the current test? Having the Web100 vars and additional
details like jitter, loss and buffer sizing you get from NDT is awfully
nice. Having an alternate methodology has value, but it would be great to
get more information. Perhaps this should be changed to be a request to
enhance the current throughput test.
